FanartHandler/FanartSelectedOther.cs

```diff
@@ -151,6 +151,16 @@ public void RefreshGenericSelectedProperties(Utils.SelectedType property, ref st
             }
             AddSelectedGenreProperty(SelectedGenre, SelectedItem, property);
 
+            // Trakt remove ClearArt for pages
+            if (SelectedItem == "Next Page" ||
+                SelectedItem == "Previous Page" ||
+                SelectedItem.StartsWith("Next ") && SelectedItem.EndsWith(" Days") ||
+                SelectedItem.StartsWith("Previous ") && SelectedItem.EndsWith(" Days"))
+            {
+              IsSelectedVideo = true;
+              Utils.SetProperty("#Trakt.Movie.ImdbId","");

```

Comment:
We cannot interfere with another plugin's operation. This is his property, only he can control it.

Reply:
Fair point, you are correct and I did not really think about that. This was to try and get the path to clear so that when the user goes to page nav then the side menu and tries to select a different list display they don't get a the last selected clearart pop up. I will need to find another way.